<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestApp For SerialPortWorker &amp; WindowProtocol &amp; BinaryProtocol</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBoxSerialPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHVonoff">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HV ON</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxHVonoff">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReadV">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReadI">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReadP">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadP</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxReadV">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxReadI">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxReadP">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COMMAND</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxHVonoff_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxReadI_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxReadV_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxReadP_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSEND">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>282</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>   SEND</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSentMsg">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSentMessage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelReadMessage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelREAD">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>412</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>   READ</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelReadMsg">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CustomTest">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CustomTest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearBuffer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearBuffer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UHVType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>UHV4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QUIT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PRIORITY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TIMES</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
